pkg/controller/registry/resolver/resolver.go

```diff
@@ -469,6 +483,19 @@ func (r *SatResolver) newSnapshotForNamespace(namespace string, subs []*v1alpha1
 		op.sourceInfo = &OperatorSourceInfo{
 			Catalog: existingOperatorCatalog,
 		}
+		for _, p := range op.properties {
```

Comment:
Sorry. I should've commented here on the question how these changes relevant to the fix are.